beacon_node/store/src/lib.rs

```diff
@@ -268,8 +268,6 @@ pub enum DBColumn {
     OptimisticTransitionBlock,
     #[strum(serialize = "bhs")]
     BeaconHistoricalSummaries,
-    #[strum(serialize = "olc")]
-    OverflowLRUCache,
```

Comment:
What's the policy to deprecate DBColumns?

Reply:
We've never done it before

We could keep it around in `DBColumn` so it can be used with `lighthouse db inspect`, and then delete it once the release that removes it is buried under a hard fork

Reply:
added it back here https://github.com/sigp/lighthouse/pull/5891/commits/8366504e7e03fb66b30923a9582b0de388a1577d